deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet

```diff
@@ -6,7 +6,7 @@
   sumologicCollector:: {
     remoteWriteConfigs+: [
       {
-        url: $._config.sumologicCollectorSvc + "prometheus.metrics.state",
+        url: "http://$(CHART)-metrics.$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.state",
```

Comment:
Do the `CHART` and `NAMESPACE` variables are going to be replaced here correctly?
No idea how the libsonnet files works, but this is kubernetes directory

Reply:
Yes, this needs to be fixed as the variables won't be replaced.